spago.dhall

```diff
@@ -1,20 +1,27 @@
-{ sources =
-    [ "src/**/*.purs", "test/**/*.purs" ]
-, name =
-    "dotenv"
+{ name = "dotenv"
 , license = "MIT"
 , repository = "https://github.com/nsaunders/purescript-dotenv.git"
 , dependencies =
-    [ "console"
-    , "effect"
-    , "node-fs-aff"
-    , "node-process"
-    , "parsing"
-    , "psci-support"
-    , "run"
-    , "spec"
-    , "sunde"
-    ]
-, packages =
-    ./packages.dhall
+  [ "aff"
```

Comment:
These dependencies were added because they are used in your source code but not explicitly listed as dependencies. Spago now throws an error if you do that, so I needed to install these.

Also, I've split your Spago file into a file for your library and other files for tests and examples. In the future the package sets will use the Spago file from your repository to determine your dependencies (rather than a separate list in the package-sets repo) so that helps you ship only the dependencies you need to.

Reply:
Terrific news, thanks @thomashoneyman !